ID-60 33across user ID documentation (#9)
33Across/prebid.github.io: dev-docs/modules/userId.md

{: .table .table-bordered .table-striped }
| Param under userSync.userIds[] | Scope | Type | Description | Example |
| --- | --- | --- | --- | --- |

> | name | Required | String | May be: `"33acrossId"`, "admixerId"`, `"qid"`, `"adtelligentId"`, `"akamaiDAPId"`, `"amxId"`, `"britepoolId"`, `"criteo"`, `"fabrickId"`, `"flocId"`, `"hadronId"`, `"id5id"`, `identityLink`, `"idx"`, `"intentIqId"`, `"justId"`, `"liveIntentId"`, `"lotamePanoramaId"`, `"merkleId"`, `"naveggId"`, `"mwOpenLinkId"`, `"netId"`, `"novatiqId"`, `"parrableId"`, `"quantcastId"`, `"pubProvidedId"`, `"sharedId"`, `"tapadId"`, `"unifiedId"`,`"uid2"`, `"verizonMediaId"`, `"zeotapIdPlus"` | `"unifiedId"`

Reviewer: ```suggestion
| name | Required | String | May be: `"33acrossId"`, `"admixerId"`, `"qid"`, `"adtelligentId"`, `"akamaiDAPId"`, `"amxId"`, `"britepoolId"`, `"criteo"`, `"fabrickId"`, `"flocId"`, `"hadronId"`, `"id5id"`, `identityLink`, `"idx"`, `"intentIqId"`, `"justId"`, `"liveIntentId"`, `"lotamePanoramaId"`, `"merkleId"`, `"naveggId"`, `"mwOpenLinkId"`, `"netId"`, `"novatiqId"`, `"parrableId"`, `"quantcastId"`, `"pubProvidedId"`, `"sharedId"`, `"tapadId"`, `"unifiedId"`,`"uid2"`, `"verizonMediaId"`, `"zeotapIdPlus"` | `"unifiedId"`
```

## User ID Sub-Modules

> ### 33Across User ID Module

Reviewer: ```suggestion
### 33Across ID
```

> The 33Across User ID Module is a way for publishers to monetize their cookieless inventory across multiple supply-side platforms via Prebid.JS. The module provides publishers with addressability for their open marketplace cookieless inventory and access to cookieless demand. The 33Across User ID Module enables publishers to match cookieless browser users to demand platforms without any reliance on traditional cookie matching methodologies. 33Across User ID Module utilizes Lexicon technology to connect Publishers to Demand partners via proprietary technologies in a probabilistic and privacy-safe manner.

Reviewer: ```suggestion
The 33Across User ID Sub-Module is a way for publishers to monetize their cookieless inventory across multiple supply-side platforms via Prebid.JS. The module provides publishers with addressability for their open marketplace cookieless inventory and access to cookieless demand. The 33Across User ID Module enables publishers to match cookieless browser users to demand platforms without any reliance on traditional cookie matching methodologies. The 33Across User ID Sub-Module utilizes Lexicon technology to connect Publishers to Demand partners via proprietary technologies in a probabilistic and privacy-safe manner.
```

> #### 33Across User ID Module Configuration

Reviewer: ```suggestion
#### 33Across ID Configuration
```

> Please make sure to add the 33across User ID module to your Prebid.js package with:

Reviewer: ```suggestion
Please make sure to add the 33across User ID sub-module to your Prebid.js package with:
```

```shell
gulp build --modules=33acrossIdSystem,userId
```

The following configuration parameters are available:
{: .table .table-bordered .table-striped }
| Param under userSync.userIds[] | Scope | Type | Description | Example |
| --- | --- | --- | --- | --- |

> | name | Required | String | The name of this module | "33acrossId" |

Reviewer: ```suggestion
| name | Required | String | The name of this sub-module | `"33acrossId"` |
```

> | params ||| Details for the module initialization ||

Reviewer: ```suggestion
| params ||| Details for the sub-module initialization ||
```

| params.pid | Required | String | Partner ID (PID) | Please reach out to [PrebidUIM@33across.com](mailto:PrebidUIM@33across.com) and request your PID |
| storage |||||

> | storage.name | Required | String | The name of the cookie or html5 local storage key | "33acrossId" (recommended) |

Reviewer: ```suggestion
| storage.name | Required | String | The name of the cookie or html5 local storage key | `"33acrossId"` (recommended) |
```

> | storage.type | Required | String | This is where the results of the 33across user ID will be stored | "html5" (recommended) or "cookie" |

Reviewer: ```suggestion
| storage.type | Required | String | This is where the 33across User ID will be stored | `"html5"` (recommended) or `"cookie"` |
```

> | storage.expires | Strongly Recommended | Number | How long (in days) the user ID information will be stored | 90 (recommended) |

Reviewer: ```suggestion
| storage.expires | Strongly Recommended | Number | How long (in days) the user ID information will be stored | `90` (recommended) |
```

> | storage.refreshInSeconds | Strongly Recommended | Number | How many seconds until the ID is refreshed | 28800 (recommended) |

Reviewer: ```suggestion
| storage.refreshInSeconds | Strongly Recommended | Number | How many seconds until the ID is refreshed | `8 * 3600` (recommended) |
```

> #### 33Across User ID Module Example

Reviewer: ```suggestion
#### 33Across ID Example
```

```
pbjs.setConfig({
  userSync: {
    userIds: [{
      name: "33acrossId",

>     params: {

Reviewer: ```suggestion
      params: {
```
